it/mail/chrome/messenger/localMsgs.properties

```diff
@@ -230,39 +230,39 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Please enter a new password for use
 # Do not translate "POP3"
 # Do not translate "%S". Place %S in your translation where the name of the server should appear.
 # Do not translate "TOP"
-4041=Il server di posta POP3 (%S) non implementa il comando TOP. Senza il supporto del server è impossibile usare le opzioni "dimensione massima del messaggio" e "scarica solo intestazioni". Queste opzioni sono state disattivate ed i messaggi saranno scaricati indipendentemente dalla loro dimensione.
+4041=Il server e-mail POP3 (%S) non implementa il comando TOP. Senza il supporto del server è impossibile usare le opzioni "Dimensioni massime del messaggio" e "Scarica solo intestazioni". Queste opzioni sono state disattivate e i messaggi saranno scaricati indipendentemente dalla loro dimensione.
 
 # secure authentication failed and unsure why
 ## @name CANNOT_PROCESS_APOP_AUTH
 ## @loc None
-4042=Il server di posta non implementa l'autenticazione sicura o è stata fornita una password non corretta. Controllare la password o disattivare l'autenticazione sicura per la posta nella finestra di impostazioni per l'account.
+4042=Il server e-mail non implementa l'autenticazione sicura o è stata fornita una password non corretta. Controlla la password o disattiva l'autenticazione sicura per le e-mail nella finestra di impostazioni per l'account.
 
 ## @name NS_ERROR_COULD_NOT_CONNECT_VIA_TLS
 ## @loc None
-4043=Impossibile stabilire un collegamento TLS al server POP3. Il server potrebbe essere inattivo o configurato non correttamente. Verificarne la configurazione nelle impostazioni dell'account e riprovare.
+4043=Impossibile stabilire un collegamento TLS al server POP3. Il server potrebbe essere inattivo o configurato non correttamente. Verifica la configurazione nelle impostazioni dell'account e riprova.
 
 ## @name POP3_AUTH_INTERNAL_ERROR
 ## @loc None
-4047=Internal state error during POP3 server authentication. This is an internal, unexpected error in the application, please report it as bug.
+4047=Errore di stato interno durante l'autenticazione al server POP3. Questo è un errore interno e imprevisto nell'applicazione, per favore segnalalo come bug.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 ## @loc None
-4048=This POP3 server does not seem to support encrypted passwords. If you just set up the account, please try changing to 'Password, transmitted insecurely' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, this is a common scenario how someone could steal your password.
+4048=Sembra che questo server POP3 non supoorti le password crittografate. Se hai appena impostato l'account, prova a selezionare 'Password, transmessa in maniera non sicura' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se prima funzionava e ora improvvisamente non funziona, questo è uno scenario comune come se qualcuno potesse rubare la tua password.
```

Comment:
CORRECTION NEEDED. 

There are two typos. Please replace "supoorti" with "supporti" and ", transmessa" with "trasmessa".